.github/scripts/download_assets.sh

```diff
@@ -22,13 +22,12 @@ download()
   ARCHIVE_NAME="$ARCHIVE_BASENAME.tgz"
   TMP_ARCHIVE_NAME="$ARCHIVE_NAME.tmp"
   TMP_DIR=$(mktemp -d)
-  URL="http://s3-de-central.profitbricks.com/xayn-yellow-bert/$NAME/$ARCHIVE_NAME"
+  URL="s3://xayn-yellow-bert/$NAME/$ARCHIVE_NAME"
 
   if [ -d "$DATA_DIR/$ARCHIVE_BASENAME" ]; then
     echo "skip downloading $DATA_DIR/$ARCHIVE_NAME"
   else
-
-    curl "$URL" -o "$DATA_DIR/$TMP_ARCHIVE_NAME" -C -
+    aws s3 cp $URL $DATA_DIR/$TMP_ARCHIVE_NAME
```

Comment:
:question: I guess this is needed because now the bucket is private

Reply:
Ok then we need to make sure everyone in the team and the CI can access the new one before we merge this.